index.html

```diff
@@ -117,8 +117,10 @@
             (via
             <a href="https://github.com/weaveworks/ignite">Weave Ignite</a>
             )
-            , and
+            ,
             <a href="https://webapp.io">webapp.io</a>
+            , and
+            <a href="https://github.com/astro/microvm.nix">microvm.nix</a>
```

Comment:
glad to see new users :) welcome to Firecracker community!

Reply:
Thank you for your review. I resolved the issues.

Reply:
thanks for the follow up!